### PR TITLE
Drop circleci, add githb actions and refators

### PR DESCRIPTION
**Pull request by cpanato**
**What type of PR is this?**
/kind feature

**Any specific area of the project related to this PR?**

/area falco-chart

**What this PR does / why we need it**:

- add dependabot config
- move to a charts directory to make it more clean
- drop circleci config
- add github actions workflows for testing and release

bonus point: it will push the oci chart to the github registry as well :)

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Checklist**

- [ ] Chart Version bumped
- [ ] Variables are documented in the README.md
- [ ] CHANGELOG.md updated

### CODE REVIEW

**Comment by cpanato** [comment]
as a follow up we can sign the chart with gpg, but will require someone to create the keys

**Comment by cpanato** [comment]
/assign @leogr @LucaGuerra 

also will need someone with admin access in circleci to remove the job from there

**Comment by cpanato** [comment]
looks like `falco-exporter` failed, checking

**Comment by cpanato** [comment]
also, the release might failed when we merge this because it see those charts as new charts. But after we merge this i will follow up to make sure everything will work

i did some tests in a sample repo and that works, and got the same issues as here

**Comment by cpanato** [comment]
for the falco-exporter maybe we ignore that for now and i will test asap we merge this

**Comment by cpanato** [comment]
now it is fixed :)

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,65 @@
+name: Release Charts
+
+on:
+  push:
+    branches:
+      - main
+      - master
+    paths:
+      - "charts/**"
```
Comment (alacuku) [inline .github/workflows/release.yml:9]: A new chart will be released each time we push changes to `charts`. Right?

Comment (cpanato) [reply .github/workflows/release.yml:9]: yes, same behavior, just blocking when there is no changes in the chart, to we be able to update other things like ci and configs

**Review by Issif — APPROVED** [review]

**Pull request merged**